flow/controllers/velocity_controllers.py

```diff
@@ -45,6 +48,10 @@ def __init__(self,
         self.d_2 = 1.0
         self.d_3 = 0.5
 
+        self.danger_edges = danger_edges if danger_edges else {}
```

Comment:
It'd be easier to set `danger_edges={}` in the function header, turning {} into the default vlaue